src/main/java/kvp/hhlin/minifullfillment/domain/Sku.java

```diff
@@ -0,0 +1,80 @@
+package kvp.hhlin.minifullfillment.domain;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Sku {
+
+    private static final int MAX_NAME_LENGTH = 20;
+
+    private String code;
+
+    private String name;
+
+    private String status;
+
+    private Set<Barcode> barcode;
```

Comment:
barcode가 여러개들어갈 수 있다면 복수형으로 네이밍을해줘도 괜찮을 것 같은데, 어떻게 생각하시나요!

```suggestion
    private Set<Barcode> barcodes;
```